packages/create-app/templates/default-app/packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -108,6 +120,10 @@ const overviewContent = (
     <Grid item md={6}>
       <EntityAboutCard variant="gridItem" />
     </Grid>
+    <Grid item md={6} xs={12}>
+      <EntityCatalogGraphCard variant="gridItem" height={400} />
+    </Grid>
```

Comment:
Just realized you added these in as well, which I think makes sense. Best mention this in the changeset as well though. Could shot this example and say that it was added to `overviewContent`, `apiPage`... etc

Reply:
Thanks, yeah adding a one-liner and one code block makes sense and will save some time :P